src/dbus_fast/message_bus.py

```diff
@@ -901,7 +908,7 @@ def _make_method_handler(
     def _find_message_handler(
         self, msg: _Message
     ) -> Optional[Callable[[Message, Callable[[Message], None]], None]]:
-        if msg.interface.startswith("org.freedesktop.DBus."):
```

Comment:
There is no optimized version of this in cython so it has to call the actual python function